### PR TITLE
Launchpad introduction

### PR DESCRIPTION
**Pull request by jansenbob**
- Add a 'Get a Launchpad' section to the docs
- Add a mention in the Quickstart Guide
- Add a blogpost

### CODE REVIEW

File: blog/2022-06-01 Introducing the Launchpad.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+<head>
+  <meta property="og:title" content="Introducing the Launchpad - a lightweight way to test Objectiv on your own app or website" />
+</head>
+
+import BlogImage from '@site/src/components/blog-image'
+import JoinSlackLink from '@site/src/components/join-slack-link';
+
+
+*Want to test Objectiv on your own app or website without worrying about the OPS part? Get a Launchpad: a free-to-use, fully managed Objectiv back-end and data store that’s ready to use for testing without any setup or configuration.*
```
Comment (ivarpruijn) [inline blog/2022-06-01 Introducing the Launchpad.md:17]: OPS is not an acronym, should just be `Ops`.

File: blog/2022-06-01 Introducing the Launchpad.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+With the introduction of the Launchpad, you can now run Objectiv on your own website or app without having to set up and manage a self-hosted Objectiv back-end & data store.
+
+
+<BlogImage url='img/blog/launchpad.svg' size="large"/>
+
+## How it works
+On request, Objectiv will set up your Launchpad and provide instructions on how to connect to it. You can then instrument Objectiv’s [tracking SDK](https://objectiv.io/docs/tracking/) to a part of your site or app to start collecting data. On average, this takes existing users a few hours.
+
+After collecting testing data, you can take any of the [example notebooks](https://objectiv.io/docs/modeling/example-notebooks/) and run them on your own data set. You can also build and run your own models by using the [Bach](https://objectiv.io/docs/modeling/bach/) modeling library.
```
Comment (ivarpruijn) [inline blog/2022-06-01 Introducing the Launchpad.md:32]: Maybe explain how you would run those notebooks? E.g. by connecting to a database endpoint we've set up for you?

File: blog/2022-06-01 Introducing the Launchpad.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Introducing the Launchpad - a lightweight way to test Objectiv on your own app or website
+description: "Want to test Objectiv on your own app or website without worrying about the OPS part? Get a Launchpad: A free-to-use, fully managed Objectiv back-end and data store that’s ready to use for testing without any setup or configuration."
```
Comment (ivarpruijn) [inline blog/2022-06-01 Introducing the Launchpad.md:3]: OPS is not an acronym, should just be `Ops`.

File: blog/2022-06-01 Introducing the Launchpad.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+slug: introducing-the-launchpad
+image: /img/open-graph/objectiv-og-large.png
```
Comment (ivarpruijn) [inline blog/2022-06-01 Introducing the Launchpad.md:5]: Let's not forget to update this one

File: docs/docs/home/get-a-launchpad.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 1
+slug: /home/get-a-launchpad
+title: Get a Launchpad
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import JoinSlackLink from '@site/src/components/join-slack-link';
+
+A Launchpad is a **fully managed Objectiv back-end and data store** that’s ready to use without any setup or configuration. Our Launchpads are free to use and enable you to quickly test-drive Objectiv on your own app or website without needing to worry about the OPS part.
```
Comment (ivarpruijn) [inline docs/docs/home/get-a-launchpad.md:10]: `OPS` --> `Ops` here as well

File: docs/docs/home/quickstart-guide.md [hunk 6]
```diff
@@ -64,6 +64,10 @@ We hope you enjoyed playing around with Objectiv. You can use `docker-compose do
 
 If you want to start using Objectiv for your own project, you'll want to set up a Collector and a data store first. [Follow this guide](/tracking/collector/getting-started.md) to get started.
 
+:::info
+Want to test Objectiv on your own app/website without setting up a Collector and data store? [Get a Launchpad](/home/get-a-launchpad.md)
```
Comment (ivarpruijn) [inline docs/docs/home/quickstart-guide.md:68]:
```suggestion
Want to test Objectiv on your own app/website without setting up a Collector and data store? [Get a Launchpad](./get-a-launchpad.md)
```

File: docs/docs/home/get-a-launchpad.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+![Objectiv Launchpad](/img/launchpad.svg "Objectiv Launchpad")
+
+### How does it work?
+
+Objectiv will set up your Launchpad and provide instructions on how to connect to it. You can then [instrument Objectiv’s tracking SDK](/tracking/) to a part of your site or app to start collecting data. On average, this takes existing users a few hours.
+
+After collecting testing data, you can take any of the [example notebooks](/modeling/example-notebooks/) and run them on your own data set. You can also build and run your own models by using the [Bach](/modeling/bach/) modeling library.
```
Comment (ivarpruijn) [inline docs/docs/home/get-a-launchpad.md:18]: See comment above: Maybe explain how you would run those notebooks? E.g. by connecting to a database endpoint we've set up for you?

File: blog/2022-06-01 Introducing the Launchpad.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+---
+title: Introducing the Launchpad - a lightweight way to test Objectiv on your own app or website
+description: "Want to test Objectiv on your own app or website without worrying about the OPS part? Get a Launchpad: A free-to-use, fully managed Objectiv back-end and data store that’s ready to use for testing without any setup or configuration."
+slug: introducing-the-launchpad
```
Comment (ivarpruijn) [inline blog/2022-06-01 Introducing the Launchpad.md:4]:
```suggestion
slug: introducing-the-launchpad/
```

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

**Comment by ivarpruijn** [comment]
LGTM

**Pull request merged**